pkg/collector/corechecks/gpu/model/model.go

```diff
@@ -7,11 +7,36 @@
 // the gpu core agent check
 package model
 
+// MemAllocType represents the possible values of memory allocation types
+type MemAllocType string
+
+const (
```

Comment:
if we don't report this breakdown for now, let's not include it in the PR. can be added in a separate PR later upon demand

Reply:
Removed from the data we send from sysprobe